test/test.js

```diff
@@ -8,7 +8,7 @@
  * Module dependencies.
  */
 var should 		= require('should');
-var OO 			= require('../index');
+var OO 			= require('../dist/index').default;
```

Comment:
I'm not sure why we're importing `.default` but anyway we should require `lib/main` due to coverage issues,